### PR TITLE
Adding amp-ima-video sample

### PR DESCRIPTION
**Pull request by shawnbuso**
This adds a sample for the amp-ima-video component.

### CODE REVIEW

File: src/20_Components/amp-ima-video.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+  The plugin provides a video player with controls for your content, and requests
+  and displays ads from a VAST-compliant ad server of your choice using the IMA SDK.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<%host%>/../components/amp-ima-video/">
```
Comment (sebastianbenz) [inline src/20_Components/amp-ima-video.html]: Are the `..` intentional in the URL? They shouldn't be needed here.

File: src/20_Components/amp-ima-video.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  <!-- ## Basic Usage -->
+  <!--
+    The amp-ima-video component HTML accepts up to two unique types of HTML nodes as children - source tags for content video, and track tags for subtitles. Both of these can be used in the same way as the standard video tag.
+
+    The component requires an ad tag, provided in data-tag, which is a URL to a VAST-compliant ad response (for examples, see IMA Sample Tags). It has an optional  data-poster attribute for a poster image to be displayed before playback.
+  -->
+  <amp-ima-video id="myVideo"
+      width="640" height="360" layout="responsive"
+      data-tag="https://pubads.g.doubleclick.net/gampad/ads?sz=640x480&iu=/124319096/external/ad_rule_samples&ciu_szs=300x250&ad_rule=1&impl=s&gdfp_req=1&env=vp&output=vmap&unviewed_position_start=1&cust_params=deployment%3Ddevsite%26sample_ar%3Dpremidpost&cmsid=496&vid=short_onecue&correlator="
+      data-poster="/img/ima-poster.jpg">
```
Comment (sebastianbenz) [inline src/20_Components/amp-ima-video.html]: Looks like the poster URL needs to be absolute: `"<%host%>/img/ima-poster.jpg"` - relative urls use ampproject.org as domain -> bug?

Comment (shawnbuso) [inline src/20_Components/amp-ima-video.html]: I was wondering what the issue was here - thanks!

File: src/20_Components/amp-ima-video.html (hunk 1 above)
Comment (shawnbuso) [inline src/20_Components/amp-ima-video.html]: Hmm not sure where these came from - removed.

**Comment by shawnbuso** [comment]
Thanks for the feedback - re: the play button not being centered, this is an bug in the extension, however I'm working towards using the video player's native controls rather than my own custom controls, which will eliminate this issue.

**Comment by sebastianbenz** [comment]
...and it's live: https://ampbyexample.com/components/amp-ima-video/

**Pull request merged**